internal/cmd/cmd.go

```diff
@@ -19,6 +19,15 @@ import (
 	"golang.org/x/sys/windows"
 )
 
+type CmdProcessRequest struct {
```

Comment:
Comment here on the reason for this. Right now you can only supply it on some specialty methods that are tailored for running a process in a uvm or on the host.